grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -528,7 +528,7 @@ class UrlMappings {
 
         "/api/replenishments/$id/"(parseRequest: true) {
             controller = { "replenishmentApi" }
-            action = [GET: "read", POST: "create", PUT: "update"]
+            action = [GET: "read", POST: "update"]
```

Comment:
I'd rather change on the frontend side, that `update` method is called with PUT, instead of changing the mapping for not correct one regarding REST standards

Reply:
sure, I guess `PUT` for an update makes much more sense than a `POST`.
I didn't want to do it since it does change the API a little bit and it seems that previously the POST method was used for update